src/utils.ts

```diff
@@ -13,12 +13,20 @@ export function convertToUnist(node: Element): GenericNode | GenericParent | und
       const { name, attributes, elements } = node;
       const children = elements?.map(convertToUnist).filter((n): n is GenericNode => !!n);
       const next: GenericNode = { type: name ?? 'unknown', ...attributes };
-      if (children) next.children = children;
+      if (name === 'code') {
+        next.value = elements?.[0].text as string;
+      } else if (children) next.children = children;
       return next;
     }
     case 'text': {
       const { attributes, text } = node;
-      return { type: 'text', ...attributes, value: String(text) };
+      return {
+        type: 'text',
+        ...attributes,
+        value: String(text)
+          .replace(/\n(\s+)$/, '')
```

Comment:
Replacing these new lines is a little bit surprising, but I guess nothing really comes from these converted trees besides printing to the console, so there fewer lines is better.

Reply:
This is actually to do with formatting of the string in pandoc. They have a strange strategy for creating indented XML. This accommodates for this.

Most XML that I have seen from publishers is not indented, it is a single line.